install/jakartaee/bin/sig-test_se8.map

```diff
@@ -143,15 +143,15 @@ jakarta.resource=2.0_se8
 
 # JAXR 1.0 
 # (This is JSR-093)
-#jakarta.xml.registry=1.0_se8
+#javax.xml.registry=1.0_se8
 
 # Java EE Management 1.1 
 # (This is JSR-077)
-#jakarta.management.j2ee=1.1_se8
+#javax.management.j2ee=1.1_se8
 
 # Java EE Deployment 1.2 
 # (This is JSR-088)
-#jakarta.enterprise.deploy=1.2_se8
+#javax.enterprise.deploy=1.2_se8
```

Comment:
Deployment is also pruned so probably should remove javax.enterprise.deploy

Reply:
Should be removed now